barretenberg/cpp/src/barretenberg/vm/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 if(NOT DISABLE_AZTEC_VM)
-  barretenberg_module(vm sumcheck)
+  barretenberg_module(vm ultra_honk stdlib_honk_verifier)
```

Comment:
why do all the native and recursive verifiers needs to be built for the avm? i wonder if you can't fine-grain the dependencies to reduce compile time

Reply:
I could reduce dependencies.